src/openforms/logging/admin.py

```diff
@@ -30,6 +75,19 @@ def has_change_permission(self, request, obj=None):
     def has_delete_permission(self, request, obj=None):
         return False
 
+    def change_view(
+        self,
+        request: HttpRequest,
+        object_id: str,
+        form_url: str = "",
+        extra_context: dict[str, Any] | None = None,
+    ) -> HttpResponse:
+        timelinelog = self.get_object(request, object_id)
+        if timelinelog is None:
+            raise Http404(f"No {self.model._meta.object_name} matches the given query.")
+        timelinelog_details_view_admin(timelinelog, request.user)
+        return super().change_view(request, object_id, form_url, extra_context)
```

Comment:
that's not what the ticket specified - the ticket specified viewing request logs from django-log-outgoing-requests

Reply:
I'm not sure this is the case. Viewing these outgoing requests logs is already available at `http://localhost:8000/admin/log_outgoing_requests/outgoingrequestslog/`.

Joeri clarified this this morning